### PR TITLE
Fix extra raster key handling

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by asaunier** [comment]
Looks good to me but what do you mean by "extra raster key handling"?

**Comment by sbrunner** [comment]
Other than mnt, mns

File: core/src/script/CGXP/plugins/ContextualData.js [hunk 1]
```diff
@@ -152,12 +152,14 @@ cgxp.plugins.ContextualData = Ext.extend(gxp.plugins.Tool, {
      */
     handleServerData: function(data) {
         var result = {};
-        Ext.each(data, function(key) {
-            if (['mnt', 'mns'].indexOf(key) < 0) {
-                result[key + '_value'] = data[key];
-                result[key + '_label'] = OpenLayers.i18n(key);
+        for (var key in data) {
+            if (data.hasOwnProperty(key)) {
+                if (['mnt', 'mns'].indexOf(key) < 0) {
```
Comment (asaunier) [inline core/src/script/CGXP/plugins/ContextualData.js]: Why not simply using the much clearer following syntax:
```
if (key != 'mnt' && key != 'mns')
```

**Comment by asaunier** [comment]
> Other than mnt, mns

OK

File: core/src/script/CGXP/plugins/ContextualData.js (hunk 1 above)
Comment (sbrunner) [inline core/src/script/CGXP/plugins/ContextualData.js]: simply ? for me it looks more complex ...

Comment (asaunier) [inline core/src/script/CGXP/plugins/ContextualData.js]: Using basic operators is easier to read. Testing among 2 values does not require using indexOf.

Comment (sbrunner) [inline core/src/script/CGXP/plugins/ContextualData.js]: It's true that with 2 value there isn't many changes but I like when it's also good form many values :-)

**Pull request merged**